gensim/test/test_similarities.py

```diff
@@ -1225,6 +1225,11 @@ def test_most_similar(self):
         second_similarities = numpy.array([similarity for term, similarity in index.most_similar(u"holiday", topn=10)])
         self.assertTrue(numpy.allclose(first_similarities ** 2.0, second_similarities))
 
+        # check propert integration with SparseTermSimilarityMatrix
```

Comment:
```suggestion
        # check proper integration with SparseTermSimilarityMatrix
```

Reply:
Found a typo. I tried to merge this in, but looks like "allow edit from maintainers" is not enabled.